src/jsdoc_transformer.ts

```diff
@@ -339,6 +339,12 @@ function createClosurePropertyDeclaration(
         prop, `Skipping unnamed member:\n${escapeForComment(prop.getText())}`);
   }
 
+  if (name === 'prototype') {
+    // It's illegal to name a property 'prototype'.
```

Comment:
Maybe expand this comment with what you wrote in the comment thread here?

Reply:
Done.